ui/src/components/mobileservices/MobileServiceView.js

```diff
@@ -31,7 +31,10 @@ class MobileServiceView extends Component {
       <React.Fragment>
         <h2 key="bound-services">Bound Services</h2>
         {this.props.boundServices && this.props.boundServices.length > 0 ? (
-          this.props.boundServices.map(service => <BoundServiceRow key={service.getId()} service={service} />)
+          this.props.boundServices.map(service => {
+            this.setDefaultBindingProperties(service);
```

Comment:
I know it is my code, but would you mind renaming setDefaultBindingProperties to addDefaultBindingProperty?  set sounds like it is changing the state of mobile service view, which it isn't

Reply:
Done